### DISCUSSION TITLE
Plover/steno background art

### DISCUSSION THREAD

**Discussion by panathea** [opensteno/plover#1474] [Ideas]
The Plover Discord can add a banner to the server and to server discovery. I was thinking that it would be nice to collect some flat-ish iconography so that we can create a little banner and have some assets for the next time we need to create steno art.

**Comment by panathea** [comment]
- The [homepage](http://www.openstenoproject.org/) at open steno has our layout and the bird mascot
- The steno key with an asterisk in it is the Plover logo
- We could include a nod to pen shorthand
- Raw notes might look okay, something like
    ```
    STPH
    SKWR
    ```
- A real steno machine

**Reply by antistic** [reply]
I quite like the background on the images in [Kaoffie's Steno Font](https://github.com/Kaoffie/steno_font), which could be behind something else as well.

**Comment by Avalai** [comment]
Put something together with Ted:

![plover_discord_banner_small](https://user-images.githubusercontent.com/6504095/150837093-4834d77a-fa5c-4400-a061-06c8982a941f.png)

Will upload the icon SVGs below.

**Reply by Avalai** [reply]
SVGs:
[plover_discord_banner.svg](https://user-images.githubusercontent.com/6504095/150838101-1c424a49-a864-4acf-a42d-aa2fc25d71bb.svg)
[steno-machine.svg](https://user-images.githubusercontent.com/6504095/150838103-174f6936-da88-47ef-885b-8b2fb35d1976.svg)
[plover-shorthand.svg](https://user-images.githubusercontent.com/6504095/150838104-f493b7fa-a7f9-4928-bdb4-881131671a5a.svg)
[georgi.svg](https://user-images.githubusercontent.com/6504095/150838105-bf636826-47e5-4bc9-9001-83e94694f668.svg)
[plover-icon-outlined.svg](https://user-images.githubusercontent.com/6504095/150838183-402c8004-c8cc-4248-be1e-607ee4c2b937.svg)
[plover_discord_icon.svg](https://user-images.githubusercontent.com/6504095/150840532-74364162-d062-4b61-9daa-cfbf9dceaab0.svg)
